configure.ml

```diff
@@ -342,8 +343,12 @@ let args_options = Arg.align [
     " Do not add debugging information in the Coq executables";
   "-profile", Arg.Set Prefs.profile,
     " Add profiling information in the Coq executables";
-  "-annotate", Arg.Set Prefs.annotate,
-    " Dumps ml annotation files while compiling Coq";
+  "-annotate", Arg.Unit (fun () -> printf "*Warning* -annotate is deprecated Please use -annot or -bin-annot instead\n"),
```

Comment:
I would add a dot at the end of both sentences, like it is done in other warnings.